### ISSUE TITLE
[COLOR]: Pittsburgh Facility - side navigation hover style contrast is too low

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#3413]
## Description
As a mouse user, I want to navigate the page and know where the interactive items are so that I may access information.

The hover background setting is `background-color:rgba(0,0,0,.05)` which is less than the [contrast requirement](https://www.w3.org/TR/WCAG21/#non-text-contrast) to distinguish a difference of hover state to indicate interactive elements.
![pitt-sidenav-light-hover-color.png](https://images.zenhubusercontent.com/5d6685f1a77bcf04870bc81c/5f8703fa-f18c-496f-a8cb-bbdc9e75489b)

## Type of Issue
Template

## Point of Contact

Jennifer

## Environment

* Operating System: all
* Browser: all

## Steps to Recreate

1. Enter `https://staging.va.gov/pittsburgh-health-care/` in browser
2. Using a mouse, move over various items in the side navigation (or elsewhere on the page)
3. Note the color of the background color for the hover state lacks sufficient contrast to meet WCAG requirements

## Tasks
Styling changes consistent with @rtwell 's design
[ ] bold 2 pixel stroke solid border box with slab for the hover state
[ ] Field has gray background contrast
[ ] vertical line for nesting subpages should be visible in hover state

![image](https://user-images.githubusercontent.com/55411834/76027072-dce87e00-5ee4-11ea-8090-ebe840ae3bb6.png)

## Acceptance Criteria

[ ] When user moves over various items in the side navigation (or elsewhere on the page) using a mouse, the hover state displays a bold solid border box with slab on the left and color contrast gray field
[ ] Vertical line for nesting subpages is visible in hover state
[ ] Contrast is adequate to meet requirements.

**Comment by rtwell** [comment]
here are my proposed changes to the left nav in staging:

1) bold solid border for the hover state

2) delete the duplicative pages (no more confusing Locations/Locations)

3) Vertical rules to help indicate child pages


![image](https://user-images.githubusercontent.com/12564977/75185713-8fb22280-5714-11ea-9e79-8ff59051c9b3.png)

**Comment by jenstrickland** [comment]
@mmiddaugh - I also noted in the Slack channel, these design changes would benefit from the VSP collaboration cycle. Since Ryan says there will be usability testing, perhaps the Pre-Usability Testing meeting?

**Comment by 1Copenut** [comment]
@rtwell Thanks for posting these up. I second Jennifer's suggestion they'd benefit from the VSP collab cycle. I'll CC the design team at the end of my comment.

I have two bits of design feedback/questions:

1. What does the `:focus` state look like on the top-level accordion buttons?
2. The highlighted **Location** doesn't read like a button that expands to unfurl actual links underneath. Am I right in assuming this is still a button, and the rest are links? If so, can we distinguish them somehow?

**Comment by rtwell** [comment]
is there an epic for the larger design convo for this? i found this one, but it's a little outdated and closed: https://github.com/department-of-veterans-affairs/va.gov-team/issues/2278

**Comment by rtwell** [comment]
i'm having trouble finding that, @jenstrickland — #2278 is what i have found

**Comment by jenstrickland** [comment]
Here is the original work ticket, for the record: https://github.com/department-of-veterans-affairs/va.gov-team/issues/1874

**Comment by mmiddaugh** [comment]
Hold on this one until design review complete

**Comment by mmiddaugh** [comment]
This issue should be resolved as part of the design changes in [#7212](https://github.com/department-of-veterans-affairs/va.gov-team/issues/7212). Will confirm once changes are available in staging.  Moving this to the Icebox until then.

**Comment by mmiddaugh** [comment]
@jenstrickland and @1Copenut this is ready for your review

**Comment by jenstrickland** [comment]
@mmiddaugh I'm not seeing the new side nav in staging or production. Is there a url I should use?

**Comment by mmiddaugh** [comment]
http://staging.va.gov/pittsburgh-health-care/

**Comment by jenstrickland** [comment]
Thank you! Validated.
